App.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
```

Comment:
please follow the import statement rules and place required imports before ui imports

Reply:
fixed it